patches/@oclif+dev-cli+1.22.0.dev.patch

```diff
@@ -100,6 +100,7 @@ index 3e613e0..8c8f698 100644
 +            'patches',
 +            'patches0',
 +            'typings',
++            'oclif.manifest.json',
```

Comment:
CI error:
```
**ERROR** Failed to apply patch for package @oclif/dev-cli
  This happened because the patch file patches/@oclif+dev-cli+1.22.0.dev.patch could not be parsed.
```
It rings a bell! `patch-package` requires patch files to be recreated. The way to do this would be:
* Revert the patch file to the `master` branch state:  
`git checkout master -- patches/\@oclif+dev-cli+1.22.0.dev.patch`
* `rm -rf node_modules/@oclif/dev-cli`
* `npm install`  (This should automatically run `patch-package` through the `postinstall` script, and patch `@oclif/dev-cli` according to the master branch patch file.)
* Edit file `node_modules/@oclif/dev-cli/lib/tarballs/build.js` file directly to add the `'oclif.manifest.json'` entry.
* Run `npx patch-package \@oclif/dev-cli` to recreate the patch file.
* `git commit` the new patch file.